__tests__/app.test.js

```diff
@@ -208,4 +208,73 @@ describe('POST /api/articles/:article_id/comments', () => {
             expect(body.msg).toBe('Bad Request')
         })
     });
+});
+describe('PATCH /api/articles/:article_id', () => {
+    test('should return 200 OK status code and respond with votes increased in the updated article', () => {
+        return db.query(`SELECT votes FROM articles WHERE article_id = 6`)
+        .then((votesBeforePatch) => {
+            const numOfVotes = votesBeforePatch.rows[0].votes
+            expect(numOfVotes).toBe(0)
```

Comment:
This is really interesting, but isnt necessary because we have created the test data so we can have the confidence in knowing that the votes are 0 for this article.